multimedia/rtsp2web/Makefile

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Unlicense
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rtsp2web
+PKG_VERSION:=0.0.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/mpromonet/rtsp2web
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=fb2fef69acd61482af1f09ce1b9e29b6735595136305e420a29a27ba8fea134c
+
+PKG_BUILD_FLAGS:=gc-sections lto
+
+LIVE555_VERSION:=2024.08.01
```

Comment:
Why this specific version and not something newer? Latest version seems to be [2025.07.19](https://download.videolan.org/pub/contrib/live555/live.2025.07.19.tar.gz).